src/main/java/address/MainApp.java

```diff
@@ -41,34 +41,38 @@
     protected UpdateManager updateManager;
     protected MainController mainController;
     protected KeyBindingsManager keyBindingsManager;
+    protected Config config;
 
     public MainApp() {}
 
     @Override
     public void init() throws Exception {
         logger.info("Initializing app ...");
         super.init();
+        initComponents();
         initConfig();
-        Config.setConfig(Config.getConfig());
         initPrefs();
         BrowserManager.initializeBrowser();
         //TODO: should this be here? looks out of place
-        initComponents();
```

Comment:
Actually I guess we could check if it is instantiated in the first place before overriding with the read configs, but it just seems weird if the initConfig method is empty in the MainApp.